spec/rspec/mocks/argument_matchers_spec.rb

```diff
@@ -295,6 +300,34 @@ module Mocks
         end
       end
 
+      describe "array_excluding" do
+        it "matches array with array_excluding different array" do
+          expect(a_double).to receive(:random_call).with(array_excluding(3, 4))
+          a_double.random_call([1, 2])
+        end
+
+        it "fails array_excluding when args aren't array", :reset => true do
+          expect(a_double).to receive(:msg).with(array_excluding(1, 2, 3))
+          expect {
+            a_double.msg(1, 2, 3)
+          }.to fail_including "expected: (array_excluding(1, 2, 3))"
+        end
+
+        it "fails array_excluding when arg doesn't contain all elements", :reset => true do
+          expect(a_double).to receive(:msg).with(array_excluding(1, 2, 3))
+          expect {
+            a_double.msg([1, 2])
+          }.to fail_including "expected: (array_excluding(1, 2, 3))"
+        end
+
+        it "fails array_excluding when using the fuzzy matcher", :reset => true  do
```

Comment:
I believe this needs a balancing example where it would match, e.g. `array_excluding(a_value > 2)`.